src/components/Reports/GDPRPermissions/SubscriberGdprPermissions.js

```diff
@@ -50,81 +50,54 @@ const SubscriberGdprPermissions = ({
-                        )}
-                      </td>
-                      <td>
-                        {field.value.toLowerCase() === 'none' ? (
-                          <div className="dp-icon-wrapper">
-                            <span className="ms-icon icon-lock dp-lock-grey"></span>
-                            <FormattedMessage id="subscriber_gdpr.value_none" />
-                          </div>
-                        ) : field.value.toLowerCase() === 'true' ? (
-                          <div className="dp-icon-wrapper">
-                            <span className="ms-icon icon-lock dp-lock-green"></span>
-                            <FormattedMessage id="subscriber_gdpr.value_true" />
-                          </div>
-                        ) : (
-                          <div className="dp-icon-wrapper">
-                            <span className="ms-icon icon-lock dp-lock-red"></span>
-                            <FormattedMessage id="subscriber_gdpr.value_false" />
-                          </div>
-                        )}
-                      </td>
-                    </tr>
-                  ))}
-                </>
-              ) : (
+    <>
+      <div>
+        <div className="dp-table-responsive">
+          {state.loading ? (
+            <Loading page />
+          ) : (
+            <table
+              className="dp-c-table dp-nested-table"
+              aria-label={_('subscriber_history.table_result.aria_label_table')}
+              summary={_('subscriber_history.table_result.aria_label_table')}
+            >
+              <thead>
                 <tr>
-                  <td>
-                    <span className="bounceIn">
-                      <FormattedMessage id="subscriber_gdpr.empty_data" />
-                    </span>
-                  </td>
-                  <td></td>
-                  <td></td>
+                  <th scope="col">
+                    <FormattedMessage id="subscriber_gdpr.permission_name" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="subscriber_gdpr.permission_description" />
+                  </th>
+                  <th scope="col">
+                    <FormattedMessage id="subscriber_gdpr.permission_value" />
+                  </th>
                 </tr>
-              )}
-            </tbody>
-          </table>
-        )}
+              </thead>
+              <tbody>
+                {state.fields.length ? (
+                  <>
+                    {state.fields.map((field, index) => (
+                      <PermissionExpandableRow field={field} key={index} email={subscriber.email} />
+                    ))}
+                  </>
+                ) : (
+                  <tr>
+                    <td>
+                      <span className="bounceIn">
+                        <FormattedMessage id="subscriber_gdpr.empty_data" />
```

Comment:
As I understand, the behavior when the are no results is different than in the mockup, right?

![image](https://user-images.githubusercontent.com/1157864/110324347-97c5c300-7ff4-11eb-96fb-d906dff5a8f2.png)

I mean, the expandable icon will be not present when there are no responses.

In my opinion, it is fine, at least at the moment, but I want to confirm if it is a decision or a missunderstunding.